src/main/java/org/sagebionetworks/bridge/udd/worker/BridgeUddProcessor.java

```diff
@@ -71,7 +70,7 @@ public void callback(String sqsMessageText) throws IOException, PollSqsWorkerBad
         String studyId = request.getStudyId();
         String startDateStr = request.getStartDate().toString();
         String endDateStr = request.getEndDate().toString();
-        LOG.info("Received request for hash[username]=" + userHash + ", study=" + studyId + ", startDate=" +
+        LOG.info("Received request for hash[username]=" + username + ", study=" + studyId + ", startDate=" +
```

Comment:
Don't log the username. This is personally identifying info. Log the hash instead.